packages/jest-config-testing-library/package.json

```diff
@@ -34,6 +35,7 @@
     "@moxy/jest-config-base": "^5.0.0"
   },
   "peerDependencies": {
+    "@testing-library/react-native": "^5.0.3",
     "jest": ">= 24.9.0 <= 26"
```

Comment:
Why is this a peer dep?

Reply:
Nevermind. Forgot this package is also used for web projects. I will remove the peer dep.

Reply:
Done ✅